docs/en/engines/table-engines/integrations/s3queue.md

```diff
@@ -13,7 +13,7 @@ This engine provides integration with [Amazon S3](https://aws.amazon.com/s3/) ec
 CREATE TABLE s3_queue_engine_table (name String, value UInt32)
     ENGINE = S3Queue(path, [NOSIGN, | aws_access_key_id, aws_secret_access_key,] format, [compression])
     [SETTINGS]
-    [mode = 'unordered',]
```

Comment:
Why does it lead to confusion? Unordered mode is a more common use case in my opinion than Ordered, so good to have it included in the example.

Reply:
I agree from a common use case perspective.
In the example, we use default values for each setting except for the mode.

Reply:
Ok